utils/build/docker/ruby/graphql23/app/controllers/system_test_controller.rb

```diff
@@ -8,11 +8,22 @@ def root
   end
 
   def healthcheck
+    version = if defined?(Datadog::VERSION)
+                if Gem.loaded_specs['datadog'].source.is_a?(Bundler::Source::Rubygems)
+                  Datadog::VERSION::STRING
+                else
+                  "#{Datadog::VERSION::STRING}-dev"
+                end
+              elsif Gem.loaded_specs['ddtrace'].source.is_a?(Bundler::Source::Rubygems)
+                DDTrace::VERSION::STRING
+              else
+                "#{DDTrace::VERSION::STRING}-dev"
+              end
```

Comment:
I have only suggestion to rewrite it into this

```suggestion
    gemspec = defined?(Datadog::VERSION) ? Gem.loaded_specs['datadog'] : Gem.loaded_specs['ddtrace']

    version = gemspec.version.to_s
    version = "#{version}-dev" unless gemspec.source.is_a?(Bundler::Source::Rubygems)
```

Reply:
Thanks ! I applied your suggestion (and used an OR to get the gemspec) in f1a9c15ed552e0709fa6e6d02c2b8d2ab76c1627